lunar-phase-simulator/package.json

```diff
@@ -28,9 +28,9 @@
     "@babel/cli": "^7.5.5",
     "eslint-plugin-security": "^1.4.0",
     "pixi.js": "~6.5.1",
-    "react": "~17.0.1",
-    "react-dom": "~17.0.1",
-    "react-range-step-input": "^1.2.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-range-step-input": "github:Evan-CTL/react-range-step-input",
```

Comment:
We should change this out to the ccnmtl package
`github:ccnmtl/react-range-step-input`